src/main/java/com/innowise/darya/service/SupplyService.java

```diff
@@ -0,0 +1,14 @@
+package com.innowise.darya.service;
+
+import com.innowise.darya.dto.SupplyDTO;
+
+
+public interface SupplyService {
```

Comment:
In case if we have only one implementation we don't have to make an interface and implementation. In such cases we can have only implementation.